### PR TITLE
[APB-2351][MP,AO] fixing several issues with error displaying and validation

### PR DESCRIPTION
**Pull request by arturopala**

### CODE REVIEW

File: conf/logback.xml [hunk 1]
```diff
@@ -43,17 +43,33 @@
         <appender-ref ref="STDOUT_IGNORE_NETTY" />
     </logger>
 
-    <logger name="com.google.inject" level="INFO"/>
+    <logger name="application" level="ERROR"/>
```
Comment (JTY009) [inline conf/logback.xml]: Should the log level be WARN?

Comment (arturopala) [reply conf/logback.xml]: nope

Comment (JTY009) [reply conf/logback.xml]: then logs regarding feature flags being turned off or on as well as whitelisting will not appear in kibana?

Comment (JTY009) [reply conf/logback.xml]: Will leave it for now. going to merge it

Comment (arturopala) [reply conf/logback.xml]: those logs have to be reported in the category starting with `uk.gov.hmrc.agentinvitationsfrontend`

Comment (JTY009) [reply conf/logback.xml]: Ok :+1:

**Pull request merged**